packages/base/links-to-many-component.gts

```diff
@@ -105,6 +106,7 @@ class LinksToManyEditor extends GlimmerComponent<Signature> {
         },
         multiSelect: true,
         createNewCard: this.cardContext?.actions?.createCard,
+        consumingCard: getContainingCard(this.args.model),
```

Comment:
this allows us to deal with a card that may be in a field that may be in a card, etc.